### ISSUE TITLE
XgBoost classifier for multi-class

### ISSUE THREAD

**Issue by ThanhNguyenRGU** [mrtrunghieu1/Multiple-Linear-Regression#1]
Please change the XgBoost classifier for multi-class by

def get_xgboost_classifier(n_classes):
    if n_classes == 2:
        return XGBClassifier(n_estimators=200, object='binary:logistic')
    else:
        return XGBClassifier(n_estimators=200, object='multi:softmax')

In Posterior and PosteriorModel, add new para n_classes for get_xgboost_classifier

def Posterior(training,label,sample,flag,n_classes)
def PosteriorModel(training,label,flag,n_classes):
